examples/configs/evaluation/hf_evaluate_config.yaml

```diff
@@ -0,0 +1,65 @@
+name: "lm-buddy-hf-evaluate"
+
+# Input dataset path
+dataset:
+  path: "s3://platform-storage/datasets/dialogsum"
+
+
+# Settings specific to the hf_evaluate entrypoint
+evaluation:
+  # metrics to be used for the evaluation
+  # (you can add "rouge", "meteor", and "bertscore" atm)
+  metrics: ["rouge", "meteor"]
+  # enable/disable tqdm to track eval progress
+  # (useful when running interactively, noisy on ray logs)
+  enable_tqdm: True
+  # rely on HF pipeline for summarization (ignored if using OAI API)
+  use_pipeline: True
+  # perform inference / evaluation on the first max_samples only
+  max_samples: 10
+  # output file path
+  # - if you provide a path complete with a filename, results will be stored in it
+  # - if you provide a dir, results will be stored in <dir>/<config.name>/eval_results.json
+  # - if you don't provide a storage path, results will be stored locally (see ~/.lm-buddy/results)
+  # storage_path: "s3://platform-storage/experiments/results/"
+
+# Model to evaluate. Choose one of the following options by uncommenting it
+
+# 1. Local model
+#    - Provide model path to load the model locally
+#    - Make sure you add quantization details (see below) if the model is too large
+#    - Optionally, add a tokenizer (the one matching the specified model name is the default)
+model:
+  path: "hf://facebook/bart-large-cnn"
+
+# # 2. OpenAI
```

Comment:
let's have one level of comments for this file

Reply:
Ok! We can also make more than one config file for the same entrypoint to show the different options, so nothing will have to be commented / decommented